packages/contracts-bedrock/test/OptimismPortal.t.sol

```diff
@@ -25,6 +25,15 @@ contract OptimismPortal_Test is Portal_Initializer {
     event Paused(address);
     event Unpaused(address);
 
+    address depositor;
+
+    function setUp() public override {
+        super.setUp();
+        depositor = makeAddr("depositor");
+        vm.deal(depositor, type(uint256).max);
+        vm.deal(address(this), type(uint256).max);
```

Comment:
Nit, but if you ever have a test that transfers ETH to one of these addresses without them first sending out ETH, you get a generic "EVM Error Revert" due to balance overflow which can be hard to track down
```suggestion
        vm.deal(depositor, type(uint128).max);
        vm.deal(address(this), type(uint128).max);
```

Reply:
This also means you'd have to change the value fuzz params to uint128. Also ok keeping this all as uint256—feel free to resolve however you like